gems/pending/appliance_console.rb

```diff
@@ -48,34 +48,6 @@
   I18n.t("advanced_settings.#{item}")
 end
 
-CANCEL        = "Cancel"
-
-# Restore database choices
-RESTORE_LOCAL   = "Local file"
-RESTORE_NFS     = "Network File System (nfs)"
-RESTORE_SMB     = "Samba (smb)"
-RESTORE_OPTIONS = [RESTORE_LOCAL, RESTORE_NFS, RESTORE_SMB, CANCEL]
-
-# Restart choices
-RE_RESTART  = "Restart"
-RE_DELLOGS  = "Restart and Clean Logs"
-RE_OPTIONS  = [RE_RESTART, RE_DELLOGS, CANCEL]
-
-# Timezone constants
-$tzdata          = nil
-TZ_AREAS         = %w(Africa America Asia Atlantic Australia Canada Europe Indian Pacific US)
-TZ_AREAS_OPTIONS = ["United States", "Canada", "Africa", "America", "Asia", "Atlantic Ocean", "Australia", "Europe",
-                    "Indian Ocean", "Pacific Ocean", CANCEL]
-TZ_AREAS_MAP     = Hash.new { |_h, k| k }.merge!(
-  "United States"  => "US",
-  "Atlantic Ocean" => "Atlantic",
-  "Pacific Ocean"  => "Pacific",
-  "Indian Ocean"   => "Indian"
-)
-TZ_AREAS_MAP_REV = Hash.new { |_h, k| k }.merge!(TZ_AREAS_MAP.invert)
-
-NETWORK_INTERFACE = "eth0"
-
```

Comment:
@jrafanie The globals were moved because they are specific to the date, time and timezone code being moved to new files/classes. The code moved down I think is just how the diff is showing it.